markdown/report.Rmd

```diff
@@ -166,121 +241,164 @@ More recently in December 2016 the ["Health and Social Care Delivery Plan"](http
 
-*	Over the past nine years, the percentage of time spent at home or in a community setting has gradually increased from 85.3% in 2010/11 to 89.2% in 2018/19.
+* In `r latest$fy`, `r max_old`s aged 85 and older spent a higher percentage of their last six months of life in the community (`r max(old$qom)`%) compared to `r min_old`s (`r min(old$qom)`%); this equates to `r max_old`s spending, on average an extra `r as.character(day_diff(max(old$qom), min(old$qom)))` days in the community in their last six months of life.
 
-*	In 2018/19, females aged 85 years and over spent a higher percentage of their last six months of life at home or in the community (90.0%) compared to males (87.8%); this equates to females spending, on average, an extra four days in the community in their last six months of life.
 
 <Div custom-style = "Heading 2_pgbrk">Results and Commentary</div>
 
-For the 53,168 residents of Scotland who died in financial year 2018/19, 89.2% of their last six months of life was spent either at home or in the community (Table 1) and 10.8% of their last six months of life was spent in hospital. This is equivalent to each individual spending an average of 20 days in hospital in the six months prior to their death (Table 1 in the [data tables](http://www.isdscotland.org/Health-Topics/Health-and-Social-Community-Care/Publications/2019-05-28/QOM_tables_201905.xls)).
+For the `r latest$deaths` residents of Scotland who died in financial year `r latest$fy`, `r latest$qom`% of their last six months of life was spent either at home or in the community (Table 1) and `r latest$qom_hosp`% of their last six months of life was spent in hospital. This is equivalent to each individual spending an average of `r round_half_up(182.5 * (latest$qom_hosp / 100))` days in hospital in the six months prior to their death (Table 1 in the [data tables](`r data_tables`)).
 
-Over the nine years, 2010/11 to 2018/19 the percentage of time spent at home or in a community setting has increased from 85.3% to 89.2%, this equates to, on average an extra seven days within the last six months of life being spent at home or in a community setting in 2018/19 compared to 2010/11 (see Figure 1 below or [Appendix A1](#AppendixA1)).
+Over the `r as.character(as.english(year(end_date) - year(start_date)))` years, `r min(basefile$fy)` to `r latest$fy` the percentage of time spent at home or in a community setting has increased from `r basefile %>% summarise_data(include_years = "first") %>% pull(qom)`% to `r latest$qom`%, this equates to, on average an extra `r day_diff(basefile %>% summarise_data(include_years = "first") %>% pull(qom), latest$qom)` days within the last six months of life being spent at home or in a community setting in `r latest$fy` compared to `r min(basefile$fy)` (see Figure 1 below or [Appendix A1](#AppendixA1)).
```

Comment:
I think we need to add as.character() for day_diff?

`as.character(day_diff(basefile %>% summarise_data(include_years = "first") %>% pull(qom), latest$qom))`

Otherwise it still shows as a number rather than English character.